recipe/meta.yaml

```diff
@@ -21,9 +21,9 @@ requirements:
     - setuptools
   run:
     - python >=3.6
-    - azure-core >=1.9.0,<2.0.0
+    - azure-core >=1.10.0,<2.0.0
     - msrest >=0.6.10
-    - azure-storage-blob >=12.6.0,<13.0.0
+    - azure-storage-blob >=12.7.0,<13.0.0
```

Comment:
https://github.com/Azure/azure-sdk-for-python/blob/8f9baf8a8e608adbc107bd5445389951f13d2a1e/sdk/storage/azure-storage-file-datalake/setup.py#L94-L98